.travis.yml

```diff
@@ -1,12 +1,24 @@
 language: python
-python:
-  - "2.6"
-  - "2.7"
-  - "3.3"
-  - "3.4"
+matrix:
+  include:
+  - python: "2.7"
+    env: TOXENV=py27
+  - python: "3.3"
+    env: TOXENV=py33
+  - python: "3.4"
+    env: TOXENV=py34
+  - python: "3.5"
+    env: TOXENV=py35
+before_script:
+  - wget "https://dist.ipfs.io/go-ipfs/v0.4.3-rc2/go-ipfs_v0.4.3-rc2_linux-amd64.tar.gz" -O /tmp/ipfs.tar.gz
+  - mkdir $HOME/bin
+  - pushd . && cd $HOME/bin && tar -xzvf /tmp/ipfs.tar.gz && popd
```

Comment:
I don't know, actually. I took the way that I did this from [the TravisCI Documentation](https://docs.travis-ci.com/user/installing-dependencies/#Installing-Projects-from-Source). Do you have a different way of doing this in mind?